app/src/main/java/org/xwiki/android/sync/activities/editcontact/EditContactPresenter.java

```diff
@@ -32,33 +32,52 @@
 import rx.Subscriber;
 import rx.android.schedulers.AndroidSchedulers;
 import rx.plugins.RxJavaPlugins;
-import rx.schedulers.Schedulers;
 import rx.subscriptions.CompositeSubscription;
 
+/**
+ * {@link BasePresenter} realisation which provide work with MVP
```

Comment:
Add few more lines to clearly get what it is doing. It acts as a middleman between the Activity and Views to provide the network responses or any other info. All the logic is mentioned here in Presenter class only.

Reply:
fixed